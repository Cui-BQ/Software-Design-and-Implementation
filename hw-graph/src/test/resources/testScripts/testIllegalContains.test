# Create graphs
CreateGraph graph

# Add 3 node to graph
AddNode graph n0

# Check contains
Contains graph null
