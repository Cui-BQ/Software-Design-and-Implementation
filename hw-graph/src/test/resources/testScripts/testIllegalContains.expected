# Create graphs
created graph graph

# Add 3 node to graph
added node n0 to graph

# Check contains
Exception: java.lang.IllegalArgumentException: node is null